### ISSUE TITLE
Undefined method '=' on basic import

### ISSUE THREAD

**Issue by cassidyclawson** [zdennis/activerecord-import#214]
Hi there,

Love this project! Much thanks to the contributors. 

I'm having trouble getting the basic import functionality to work using primitive arrays with Rails/Active Record 4.2.5 and activerecord-import 0.10.0.

It's my goal to routinely insert hundreds of records for a individual users. Here is some basic test code that fails:

```ruby
        user_ids = Array.new(5) { "232" }
        numbers = ["hi", "hello", "test", "whatever", "yo"]

        columns = [:user_id, :phone]
        values = [user_ids, numbers]

        Contact.import columns, values, validations: false
```

I get the following error:

```
NoMethodError (undefined method `=' for #<Contact:0x007fc4c69d5840>):
  app/controllers/api/v1/users_controller.rb:62:in `contacts'
```

The contacts model is totally basic:
```class Contact < ActiveRecord::Base
  belongs_to :user
end```

```  
create_table "contacts", force: :cascade do |t|
    t.integer  "user_id",    null: false
    t.string   "phone",      null: false
    t.datetime "created_at"
    t.datetime "updated_at"
  end

  add_index "contacts", ["phone"], name: "index_contacts_on_phone", using: :btree
  add_index "contacts", ["user_id"], name: "index_contacts_on_user_id", using: :btree
```

Any input or advice would be much appreciated.

**Comment by jkowens** [comment]
@cassidyclawson I don't think your `values` array is formatted correctly. Each child array should represent an individual record per the [wiki](https://github.com/zdennis/activerecord-import/wiki/Examples#import-using-columns-and-arrays).

I think the following assignment for `values` would make your example work:

`values = user_ids.map.with_index { |id, index| [id, numbers[index]] }`

**Comment by zdennis** [comment]
I believe @jkowens is spot on as `values` needs to be an array of arrays which each sub-array representing a value-set for one object/record.

**Comment by cassidyclawson** [comment]
Sorry for the delay in response. Thank you so much for the help! Keep up the great work.

Cassidy Clawson
(831) 247-3750




On Fri, Jan 15, 2016 at 1:52 PM -0800, "Zach Dennis" <notifications@github.com> wrote:












Closed #214.
